Makefile

```diff
@@ -133,13 +134,24 @@ install-linters: ## Install linters
 
 install-deps-libc: configure-build ## Install locally dependencies for testing libskycoin
 	git clone --recursive https://github.com/skycoin/Criterion $(BUILD_DIR)/usr/tmp/Criterion
-	mkdir $(BUILD_DIR)/usr/tmp/Criterion/build
+	mkdir -p $(BUILD_DIR)/usr/tmp/Criterion/build
 	cd    $(BUILD_DIR)/usr/tmp/Criterion/build && cmake .. && cmake --build .
 	mv    $(BUILD_DIR)/usr/tmp/Criterion/build/libcriterion.* $(BUILD_DIR)/usr/lib/
 	cp -R $(BUILD_DIR)/usr/tmp/Criterion/include/* $(BUILD_DIR)/usr/include/
 
+install-googletest-libc: configure-build ##Install googletest in debian && ubuntu
```

Comment:
why ? isn't it using `check` now ? why installing Google Test ?

Reply:
That is already fixed in `simelo:stdevHan_t11_device_build_matrix`